create-oci.sh

```diff
@@ -19,16 +19,12 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-tar_opts=-mczf
+tar_opts=(--touch --create --use-compress-program='gzip -n' --file)
```

Comment:
Shouldn't the comment for the -n on the gzip be moved here as well?

Reply:
Thanks! Added the comment, I also noticed that `--touch` only makes sense when extracting so I removed it.